.zprofile

```diff
@@ -130,7 +130,7 @@ ex=🎯:\
 
 [ ! -f ${XDG_CONFIG_HOME:-$HOME/.config}/shortcutrc ] && shortcuts >/dev/null 2>&1 &
 
-if pacman -Qs libxft-bgra >/dev/null 2>&1; then
+if [ "$WM" != "dwm" ] || pacman -Qs libxft-bgra >/dev/null 2>&1 ;then
```

Comment:
Why exclude dwm?

Reply:
Bcoz other WMs won't require libxft check.

Reply:
I'm pretty sure you need libxft-bgra for st font rendering, it isn't a window manager issue?

Reply:
Yeap I've totally forgotten that... thanks for mentioning.